docs/advanced-features/output-file-tracing.md

```diff
@@ -18,7 +18,26 @@ Next.js' production server is also traced for its needed files and output at `.n
 
 To leverage the `.nft.json` files emitted to the `.next` output directory, you can read the list of files in each trace which are relative to the `.nft.json` file and then copy them to your deployment location.
 
+## Automatically Copying Traced Files (experimental)
+
+Next.js can automatically create a `standalone` folder which copies only the necessary files for a production deployment including select files in `node_modules`.
+
+To leverage this automatic copying you can enable it in your `next.config.js`:
+
+```js
+module.exports = {
+  experimental: {
+    outputStandalone: true,
+  },
+}
+```
+
+This will create a folder at `.next/standalone` which can then be deployed on it's own without installing `node_modules`.
+
+To make leveraging this output easier a minimal `server.js` file is also output which can be used instead of `next start`. This minimal serer does not handle serving the `static` directory as this should be handled by a CDN instead.
```

Comment:
```suggestion
Additionally, a minimal `server.js` file is also output which can be used instead of `next start`. This minimal server does not handle serving the `static` directory as this should be handled by a CDN instead.
```